libs/deal-ticket/src/components/deal-ticket-validation/use-order-validation.tsx

```diff
@@ -45,6 +46,15 @@ export const marketTranslations = (marketState: MarketState) => {
   }
 };
 
+export type DealTicketSection =
```

Comment:
suggestion (non blocking): I think its a touch cleaner to export a constant object then you could dynamically create this type based on the object values

Reply:
not quite get it how such an object could look like, and at least what would be an advantage of creating an object instead a list of constants. So if it's non blocking I leave it as it is